content/doc/book/installing/index.adoc

```diff
@@ -149,7 +149,7 @@ any operating system or platform that supports Java.
 *To download and run the WAR file version of Jenkins:*
 
 . Download the
-  link:http://mirrors.jenkins.io/war-stable/latest/jenkins.war[latest stable
+  link:https://get.jenkins.io/war-stable/latest/jenkins.war[latest stable
```

Comment:
This one is [not served by any mirror](https://get.jenkins.io/war-stable/latest/jenkins.war?mirrorlist).  Can it be reworked to use the specific version number so that the mirroring works?

Reply:
Ah oops I’ll fix